controllers/ManagementController.php

```diff
@@ -140,7 +165,7 @@ private function checkIfPermissionExists($id)
         if (!is_null($userPermission = SocialMediaPermission::findOne(['publisher_id' => $id]))) {
             return $userPermission;
         }
-        return null;
+        return $userPermission;
     }
```

Comment:
This function is useless, it always returns the result of SocialMediaPermission::findOne